cmd/version.go

```diff
@@ -8,26 +8,38 @@ import (
 )
 
 func versionCmd() *cobra.Command {
-	return &cobra.Command{
+	var silent bool
+
+	command := cobra.Command{
 		Use:   "version",
 		Short: "Print version/build info",
 		Long:  "Print version/build information",
 		Run: func(cmd *cobra.Command, args []string) {
-			printVersion()
+			printVersion(silent)
 		},
 	}
+
+	command.PersistentFlags().BoolVarP(&silent, "short", "s", false, "Simplified print version for resumed printing")
```

Comment:
@brunohms Small detail - let's call the variable `short` vs `silent` and change the description to something like `Prints K9s version info in short format '

Reply:
Changed